dallinger/recruiters.py

```diff
@@ -366,7 +366,7 @@ def open_recruitment(self, n: int = 1) -> dict:
 
         return {
             "items": [study_info["external_study_url"]],
-            "message": "Study now published on Prolific",
```

Comment:
What about making it dependent on the value of `auto_recruit`?

Reply:
I don't think this particular function ever publishes the experiment on Prolific, so I think logging that here would be confusing? I think this is done implicitly by the recruit function? I'm not really sure though.